mrjob/cloud.py

```diff
@@ -77,9 +77,6 @@ class HadoopInTheCloudJobRunner(MRJobBinRunner):
         'zone',
     }
 
-    # so far, every service provides the ability to run bootstrap scripts
-    _BOOTSTRAP_MRJOB_IN_PY_FILES = False
```

Comment:
Why is this removed?

Reply:
Because installing mrjob on the cluster at bootstrap time no longer helps when we're using Docker, since that won't impact what libraries are available inside the Docker container.